README.md

```diff
@@ -2,14 +2,21 @@
 
 [![Build Status](https://travis-ci.org/axa-ch/patterns-library.svg?branch=develop)](https://travis-ci.org/axa-ch/patterns-library)
 
-[Pattern Library Preview](https://patterns.axa.ch)
+## [>> Pattern Library](https://patterns.axa.ch)
 
 [Check out our Contribution File](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#rules-and-lintings)
 [Check out our Architecture File](https://github.com/axa-ch/patterns-library/blob/develop/ARCHITECTURE.md)
 [Check out our Code of Conduct](https://github.com/axa-ch/patterns-library/blob/develop/CODE_OF_CONDUCT.md)
 
-- We only export holistic and self contained WebComponents based on CustomElement, ShadowDom and HTML Templates.
-- We do not support Built-ins
+## Communication
+
+Use the following channels for different kind of requests/reports:
+- Bug, small change, "wish": https://github.com/axa-ch/patterns-library/issues
+- Questions, ask for help, request presentations, etc: Slack #patterns-lib-devs
+- Feature requests: Slack @martin.stuedle
```

Comment:
I would say: components requests instead of feature

Reply:
@LucaMele @raphaellueckl 
Awesome to add these channels for communication.

Would you agree to add this information also inside an Github Issue template, so that every time if someone wants to create an issue and somehow missed this info, he will get it directly if he creates a new issue?
https://help.github.com/en/articles/creating-issue-templates-for-your-repository

Reply:
@AndyOGo A very neat idea! This will maybe help some people that don't know better when they start an issue. This would lead to a cleaner workflow.

Reply:
@LucaMele I think "Component Request" is a subset of "Feature". I would keep it as simple as possible. Otherwise we might get other big request and the complaint "But it is not a component what we request!! >:-() "

Reply:
Ok good point